symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,9 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if (($trustedProxies = getenv('TRUSTED_PROXIES')) && ($trustedHeaders = getenv('TRUSTED_HEADERS'))) {
+    Request::setTrustedProxies(explode(' ', $trustedProxies), constant(Request::class.'::'.$trustedHeaders));
```

Comment:
I suggest using JSON instead. It's more standard, more reliable and in sync with `symfony/symfony#23901` (it has a `json` prefix but not a `csv` one).